src/Containers.jsx

```diff
@@ -50,7 +50,7 @@ const ContainerActions = ({ con, container, onAddNotification, localImages, upda
     const isRunning = container.State.Status == "running";
     const isPaused = container.State.Status === "paused";
 
-    const deleteContainer = (event) => {
+    const deleteContainer = (_event) => {
```

Comment:
Similarly here, let's just make this `()`

Reply:
Cleaned up everywhere.